ci-operator/step-registry/optional-operators/configure-proxy-registry/test.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operator.openshift.io/v1alpha1
```

Comment:
I think this will fail validation

Reply:
@stevekuznetsov 
Is it the problem with API version ?
if so which version would you suggest.

Reply:
As in the file is not needed and can be removed.